thundra/thundra_agent.py

```diff
@@ -142,70 +146,66 @@ def wrapper(event, context):
 
     call = __call__
 
-    def start_debugger_tracing(self):
+    def initialize_debugger(self):
         try:
             import ptvsd
-            ptvsd.enable_attach(address=("localhost", config.debugger_broker_port()))
-
-            self.debugger_process = subprocess.Popen([
-                "/opt/socat",
-                "TCP:localhost:{}".format(config.debugger_broker_port()),
-                "TCP:{}:{}".format(config.debugger_broker_host(), config.debugger_broker_port())]
-              )
-
-            ptvsd.wait_for_attach(config.debugger_max_wait_time()/1000)
-            self.wait_for_debugger()
-            ptvsd.tracing(True)
-
+            ptvsd.enable_attach(address=("localhost", config.debugger_port()))
         except Exception as e:
             logger.error("error while setting tracing true to debugger using ptvsd: {}".format(e))
 
 
-    def wait_for_debugger(self):
-        start = time.time()
-        prev_rchar = 0
-        prev_wchar = 0
-        while time.time() - start < config.debugger_max_wait_time()/1000:
-            try:
-                io_metrics = self.get_debugger_proxy_io_metrics()
-                if not io_metrics:
-                    time.sleep(1)
-                    break
-                if prev_rchar != 0 and prev_wchar != 0 and ( io_metrics[0] == prev_rchar and io_metrics[1] == prev_wchar):
+    def start_debugger_tracing(self):
+        try:
+            import ptvsd
```

Comment:
I think,  if importing `ptvsd` fails it might be better to print a warning message by saying that Thundra ptvsd layer must be added